src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.impl;
+
+import database.Storage;
+import java.util.List;
+import service.ReportService;
+import service.TransactionService;
+
+public class ReportServiceImpl implements ReportService {
+
+    @Override
+    public void createReport(List<TransactionService> transactionServiceList, Storage storage) {

```

Comment:
```suggestion
    public void processTransaction(List<Transaction> transactions, Storage storage) {
```
Don’t parametrize the List by service use a simple model. This method don't  have any relation to the report so this name isn’t  appropriate